traffic_ops/traffic_ops_golang/crconfig/servers.go

```diff
@@ -190,10 +191,12 @@ select s.host_name, ds.xml_id
 from deliveryservice_server as dss
 inner join server as s on dss.server = s.id
 inner join deliveryservice as ds on ds.id = dss.deliveryservice
+inner join type as dt on dt.id = ds.type
```

Comment:
Nitpick: an inline `and ds.type <> (select id from type where name = " + string(tc.DSTypeAnyMap) + ")` is faster than a join, because the query optimizer can't know beforehand that there's only one row.

Reply:
Interesting...I'm trying to understand that. Can you elaborate a bit more please? What do you mean that there's only one row? Do you mean one column -- `type.name`?